resources/qml/ProfileOverview.qml

```diff
@@ -0,0 +1,50 @@
+//Copyright (c) 2022 Ultimaker B.V.
+//Cura is released under the terms of the LGPLv3 or higher.
+
+import Qt.labs.qmlmodels 1.0
+import QtQuick 2.7
+import QtQuick.Controls 1.4 as OldControls
+import QtQuick.Controls 2.15
+
+import UM 1.5 as UM
+import Cura 1.6 as Cura
+
+Cura.TableView
+{
+    id: profileOverview
+
+    property var qualityItem //The quality profile to display here.
+    property int extruderPosition: -1 //The extruder to display. -1 denotes the global stack.
+
+    property bool isQualityItemCurrentlyActivated:
```

Comment:
Propose to maybe rewrite to:

~~~
property bool isQualityItemCurrentlyActivated: qualityItem != null && qualityItem.name == Cura.MachineManager.activeQualityOrQualityChangesName
~~~

Reply:
This code was moved from the deleted `ProfileTab.qml` file (lines 17 to 24). It wasn't written or modified in this PR.

I do agree though that this is more concise and easier to read (sometimes that style of having brackets on separate lines is just stupid). I can change it, since it has practically no risk.